dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/net/info/BroadcastReceiverNetworkInfoProvider.kt

```diff
@@ -101,13 +101,40 @@ internal class BroadcastReceiverNetworkInfoProvider :
             NetworkInfo(
                 connectivity,
                 carrierName.toString(),
-                carrierId
+                carrierId,
+                cellularTechnology = getCellularTypeName(subtype)
```

Comment:
I could not see any changes in the unit tests related with this new logic/field.